app/controllers/teaching_classes_controller.rb

```diff
@@ -68,6 +70,18 @@ def find_teaching_class
     render_failed_response("Record not found for ID #{params[:id]}")
   end
 
+  def assign_subjects_to_class
```

Comment:
@josephelias-rubyians This needs to be written as create API for the intermediate table teaching_class_subjects.